### PR TITLE
Mob Spawner Rework

### PR DESCRIPTION
**Pull request by FalloutFalcon**
## About The Pull Request

reworks most mob spawners to fall under a much simpler spawner/burrow
each planet has one simple nest instead of the old system with difficultly levels i never even noticed were there despite the fact i used to basically just kill nests and powergame when i first played
this has the byproduct of killing off all the awful magic item spawns

attacking a bunch of other cruft, mostly lavaland

## Changelog

:cl:
add: A new generic nest used across all planets
del: Old portal and tendrils and other cruft
refactor: refactorted mob spawners to have one for each planet
/:cl:

### CODE REVIEW

**Comment by Sun-Soaked** [comment]
poke poke (fnish the atomization of necro disease & crusher stuff)

File: code/modules/mob_spawner/spawner.dm [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+	move_resist = INFINITY
+	anchored = TRUE
+	density = TRUE
+
+	var/max_mobs = 5
+	var/spawn_time = 300 //30 seconds default
+	var/mob_types = list(/mob/living/simple_animal/hostile/carp)
+	var/spawn_text = "emerges from"
+	var/faction = list("hostile")
```
Comment (Mirag1993) [inline code/modules/mob_spawner/spawner.dm:15]: why hostile?

Comment (FalloutFalcon) [reply code/modules/mob_spawner/spawner.dm:15]: This was not even my change. Most of the subtypes that matter use `"mining"`

Comment (Mirag1993) [reply code/modules/mob_spawner/spawner.dm:15]: > This was not even my change. Most of the subtypes that matter use `"mining"`

That's the problem: the mobs go full Highlander mode — killing each other until there can be only one.

**Pull request merged**